src/solvers/dgsem_structured/dg.jl

```diff
@@ -76,6 +76,14 @@ end
     end
 end
 
+@inline function get_inverse_jacobian(inverse_jacobian,
+                                      mesh::Union{StructuredMesh, StructuredMeshView,
+                                                  UnstructuredMesh2D, P4estMesh,
+                                                  T8codeMesh},
+                                      element, indices...)
+    return inverse_jacobian[indices..., element]
```

Comment:
```suggestion
                                      indices...)
    return inverse_jacobian[indices...]
```
Maybe like this to use the usual ordering `i, j, element` that we also use in `get_node_vars` etc.? The other places would have to be adapted similarly.

Reply:
I made the swap of the arguments. I am not sure why all the tests failed.

Reply:
The `...`  need to be the last arguments of a function.

Reply:
I think Hendriks suggestion was to just delete the argument `element` and let `element` be simply one of the arguments passed as `indices` to the function.

Reply:
I believe then you also don't need to dispatch on the mesh type since in both cases the function just returns `inverse_jacobian[indices...]` (where `indices` is of different length depending on the mesh type).

Reply:
Ah, but this doesn't work because you want to call the `TreeMesh` function also with `i`, `j` and `k`...

Reply:
Hmm, okay. Should I just revert this then to have `element` come last?

Reply:
Right, here `indices...` should only mean the `i`, `j`, or `k` index. We need to keep the `element` index separate because the `TreeMesh` stores things differently than all the other mesh types.

Reply:
For the tree mesh, you would do `element = last(indices)`